incubator/java-openliberty/stack.yaml

```diff
@@ -22,4 +22,5 @@ requirements:
     appsody-version: ">= 0.5.0"
 templating-data:
     libertyVersion: '19.0.0.12'
+    libertyMavenPluginVersion: '3.2'
```

Comment:
So this only shows up in a single place in the project?   A project POM would be enough to avoid duplication.   I guess you're viewing this as documentation, or bubbling up into one place.   But for something so POM-specific anyway ( a plugin version), maybe a parent POM is better?   Not totally sure....have I made you think more?

Reply:
My thinking with this was to keep these values one place, but I agree, this might be over complicating things. I can move this to just be defined in the parent pom.